File: lab/lineages/candidate-40__tacit/fixtures/box.py
def greet(name, greeting="hi", times=1):
    return greeting * times


class Box:
    def __init__(self, width, height=10, label="box"):
        self.width = width


from dataclasses import dataclass


@dataclass
class Cfg:
    timeout: int = 30
    retries: int = 3


def use():
    greet("x")
    greet("x", "yo")
    greet("x", times=2)
    Box(1)
    Box(1, height=10)
    Box(1, height=20, label="crate")
    Cfg()
    Cfg(timeout=30)
    Cfg(timeout=5)

File: lab/lineages/destroyer-14__tacit/fixtures/overload/box.swift
public struct Box {
    public init(width: Double = 10) {}
    public init(width: String = "10") {}
}
public func seed() {
    _ = Box()
    _ = Box(width: 10)
    _ = Box(width: "10")
    _ = Box(width: 20)
}
